tests/crds.yaml

```diff
@@ -0,0 +1,200 @@
+---
```

Comment:
Hm, I don't think we want to check this in. Don't we pull this from libcalico each CI run?

Reply:
Woops yeah this was unintended